dbms/src/Functions/array/registerFunctionsArray.cpp

```diff
@@ -33,7 +33,7 @@ void registerFunctionArrayDistinct(FunctionFactory & factory);
 void registerFunctionArrayFlatten(FunctionFactory & factory);
 void registerFunctionArrayWithConstant(FunctionFactory & factory);
 void registerFunctionArrayZip(FunctionFactory & factory);
-
+void registerFunctionAUC(FunctionFactory &);
```

Comment:
It is probably better to use an `array-` name for an array function.
Also, are there some longer names to consider?

Reply:
Already renamed "auc" to "arrayAUC".